Resources/Prototypes/Entities/Mobs/NPCs/space.yml

```
@@ -167,6 +167,10 @@
       collection: FootstepBounce
   - type: ReplacementAccent
     accent: kangaroo
+  - type: Butcherable
+    spawned:
+    - id: FoodMeat
+    - amount: 1
```

There shouldn't be a dash before amount?

```suggestion
    - id: FoodMeat
      amount: 1
```